Dockerfile

```
@@ -12,27 +12,15 @@ RUN apk add --no-cache \
       libffi-dev \
       libxslt-dev \
       openldap-dev \
-      postgresql-dev
-
-WORKDIR /install
-
-RUN pip install --prefix="/install" --no-warn-script-location \
-# gunicorn is used for launching netbox
-      gunicorn \
-      greenlet \
-      eventlet \
-# napalm is used for gathering information from network devices
-      napalm \
-# ruamel is used in startup_scripts
-      'ruamel.yaml>=0.15,<0.16' \
-# django_auth_ldap is required for ldap
-      django_auth_ldap \
-# django-storages was introduced in 2.7 and is optional
-      django-storages
+      postgresql-dev \
+      python3-dev \
+      py3-pip
 
 ARG NETBOX_PATH
-COPY ${NETBOX_PATH}/requirements.txt /
-RUN pip install --prefix="/install" --no-warn-script-location -r /requirements.txt
+COPY ${NETBOX_PATH}/requirements.txt requirements-container.txt /
+RUN python3 -m venv /opt/netbox/venv
+RUN /opt/netbox/venv/bin/python3 -m pip install --upgrade pip setuptools
```

I guess I would combine these lines into one, reducing the layers in the resulting Docker Image:

```suggestion
RUN python3 -m venv /opt/netbox/venv
  &&  /opt/netbox/venv/bin/python3 -m pip install --upgrade pip setuptools
```

We should probably even combine these with `RUN apk add --no-cache \`, as these two command have no dependency on `COPY ${NETBOX_PATH}/requirements.txt requirements-container.txt /`.

---

This is only in the builder image. So it should make no difference for the final image. But it will probably make the build a litte bit faster.